xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -105,7 +105,7 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notification
   [windowMenu addItem:fullscreenMenuItem];
   [windowMenu addItemWithTitle:@"Float on Top"
                         action:@selector(floatOnTopToggle:)
-                 keyEquivalent:@"t"];
+                 keyEquivalent:@""];
```

Comment:
why did you decide to remove shortcut?

Reply:
because no other app that I've seen has a shortcut for this.
Also we are not trapping the cmd+t to pipe that to the windowing system and we also lack messages for this. Considering the relevance of the functionality and the additional complexity on other parts of the code I think it is better to remove the shortcut.
I've left keyEquivalentModifierMask which is a leftover, will fix that.

Reply:
> because no other app that I've seen has a shortcut for this.

not many apps have such menu item in the first place :)

> Also we are not trapping the cmd+t to pipe that to the windowing system and we also lack messages for this. 

sorry for stupid question, but is trapping needed if we handle the menu click in the selector?

Reply:
> not many apps have such menu item in the first place :)

Some actually have but with a different name (like "keep on top" on notes)

> sorry for stupid question, but is trapping needed if we handle the menu click in the selector?

No that will simply invoke the selector. The shortcut will also work if you have the app bar focused but if Kodi is the key window we'd have to trap the shortcut (just like we do for fullscreen) and somehow feed a message into the windowing system. This is something quite specific to macOS (not available in any of the other windowing systems) and imho it's just increased complexity for no real meaningful advantage.
But if you request it really hard I might have an attempt to implement the shortcut :)

Reply:
Now it's clear, thanks!

The hard way is not required.